apps/web/src/hooks/use-export-collectible.ts

```diff
@@ -26,6 +24,12 @@ export function useExportCollectible(passphrase: string) {
   const [selectedAccount, selectAccount] = useState('')
   const [exportStatus, setExportStatus] = useState<ExportStatus>('idle')
   const connectorReference = useRef<IConnector>()
+  const Environment = useConfig()
+
+  const algorand = useMemo(
```

Comment:
I didn't test this but figure we can cross this bridge when we come back to it. LMK if anything seems off though.